requirements.txt

```diff
@@ -0,0 +1,116 @@
+ansicolors==1.1.8
+argcomplete==2.0.0
+attrs==22.1.0
+awscli==1.27.53
+bcrypt==3.2.2
+Beaker==1.10.0
+beautifulsoup4==4.11.2
+blivet==3.5.0
+blivet-gui==2.4.0
+botocore==1.29.53
```

Comment:
Should we really have all these in requirements.txt, feel like it is all the sub dependencies of gensim etc. Only gensim and ansi-colors should be enough or?

Reply:
They need to be installed either way and this doesn't hurt that. Better to be sure in this moment